### PR TITLE
fixed test_action fail

### PR DESCRIPTION
**Pull request by athackst**
Was getting "recalled" state.  Need to wait between creating client and canceling.  If aciton server still thinks it's pending, it sets the state to "recalled" instead of "canceled"

### CODE REVIEW

**Comment by tinawright** [comment]
would make the test accept recalled or cancelled instead work to test what you want here? generally having timing dependencies in tests make CI flaky, which has cascading consequences when we add  features like push-on-green.

File: test/test_action.py [hunk 1]
```diff
@@ -170,6 +170,8 @@ def test_cancel(self):
         result = ac.tick()
         assert_equal(result.status, NodeStatus.ACTIVE)
 
+        rospy.sleep(0.5)
```
Comment (LauraStelzner-TRI) [inline test/test_action.py]: Is there anyway we can use the simulation of time during the test?

**Comment by athackst** [comment]
Good point.  I'm also not a fan of sleeps. Updated.

**Review by LauraStelzner-TRI — APPROVED** [review]
LGTM

**Review by tinawright — APPROVED** [review]
awesome, thanks!

**Pull request merged**